tools/deployment/make_linux_package.sh

```diff
@@ -13,11 +13,11 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SOURCE_DIR="$SCRIPT_DIR/../.."
 BUILD_DIR=${BUILD_DIR:="$SOURCE_DIR/build/linux"}
 
-export PATH="$PATH:/usr/local/bin"
+export PATH="/usr/local/osquery/bin:/usr/local/bin:$PATH"
 source "$SOURCE_DIR/tools/lib.sh"
 
 PACKAGE_VERSION=`git describe --tags HEAD || echo 'unknown-version'`
-PACKAGE_ARCH=`uname -m`
+PACKAGE_ARCH="x86_64"
```

Comment:
If we want this to be always be `x86_64` maybe we should compare it to `uname -m` and if it does not match, abort.